api/token.go

```diff
@@ -195,9 +195,21 @@ func generateAccessToken(user *models.User, expiresIn time.Duration, config *con
 	var tigrisClaims = make(map[string]interface{})
 	// superadmin doesn't have app metadata
 	if user.AppMetaData != nil {
-		tigrisClaims = map[string]interface{}{
-			"nc": user.AppMetaData.TigrisNamespace,
-			"p":  user.AppMetaData.TigrisProject,
+		// while we migrate to RBAC - put a check
+		if user.AppMetaData.Roles != nil && len(user.AppMetaData.Roles) > 0 {
```

Comment:
Just `if len(user.AppMetaData.Roles) > 0 {`
Length of nil slice is zero.

Reply:
Thanks.  updated.